src/app/(nextjs_migration)/components/client/UserMenu.tsx

```diff
@@ -28,19 +28,20 @@ export const UserMenu = ({ session, fxaSettingsUrl }: Props) => {
 
   return (
     <div className="user-menu-wrapper" tabIndex={-1}>
-      <Script src="/nextjs_migration/client/js/userMenu.js" />
+      <Script type="module" src="/nextjs_migration/client/js/userMenu.js" />
       <button
         aria-expanded="false"
         aria-haspopup="true"
         className="user-menu-button"
         title={l10n.getString("menu-button-title")}
       >
-        <Image
+        {/* The avatar is an SVG, which next/image doesn't process: https://nextjs.org/docs/pages/api-reference/components/image#dangerouslyallowsvg */}
```

Comment:
question: should we refactor this tag to display an SVG file directly, instead of within an image/src?

Reply:
The challenge is that the image is hosted by FxA, so we don't really know what it's going to look like.